ci/install.sh

```diff
@@ -4,11 +4,10 @@ main() {
     local tag=$(git ls-remote --tags --refs --exit-code \
                     https://github.com/rust-lang-nursery/mdbook \
                         | cut -d/ -f3 \
-                        | grep -E '^v[0.1.0-9.]+$' \
+                        | grep -E '^v[0-9\.]+$' \
```

Comment:
Also, this regex didn't work as I assume it was meant to.

If we'd like to constrain to a major version, we should replace this with:

```sh
grep -E '^v0.3.[0-9]+$'
```

Reply:
Oh, this implicitly takes us from 0.2.x to 0.3.x. We may want to see if this has negative implications. @steveklabnik this may need to introduce a third version to the upstream book version hack.